src/document/DocumentCommandHandlers.js

```diff
@@ -528,6 +530,7 @@ define(function (require, exports, module) {
         return handleFileOpen(commandData).done(function (file) {
             var paneId = (commandData && commandData.paneId) || MainViewManager.ACTIVE_PANE;
             MainViewManager.addToWorkingSet(paneId, file, commandData.index, commandData.forceRedraw);
+            HealthLogger.fileOpened(file.fullPath, true);
```

Comment:
Yes it is meant to track twice, one log tracks all files opened to working set, and another the total of all files opened.